packages/documentation/src/stories/utilities/vertical-align/vertical-align.styles.scss

```
@@ -0,0 +1,59 @@
+.vertical-align-example {
+  border: 1px solid rgb(204, 204, 204);
+  line-height: 3;
```

We only want to have these styles applied on the non-table stories.

```suggestion
  :not(:has(table))& {
    border: 1px solid rgb(204, 204, 204);
    line-height: 3;
  }
```